src/util/subprocess.h

```diff
@@ -1009,27 +955,12 @@ class Popen
     execute_process();
   }
 
-/*
-  ~Popen()
-  {
-#ifdef __USING_WINDOWS__
-    CloseHandle(this->process_handle_);
-#endif
-  }
-*/
-
   int pid() const noexcept { return child_pid_; }
 
   int retcode() const noexcept { return retcode_; }
 
   int wait() noexcept(false);
 
-  int poll() noexcept(false);
```

Comment:
ed39d261dc056e700b79da769e085c8af389451d:

The `Popen::poll()` function is used in #29868. I hope that an improved/fixed Windows implementation of the `Popen::retcode()`, which I don't have now, will allow to drop it.